tests/utils.js

```diff
@@ -21,6 +21,12 @@ export function getCss(style) {
       .join('')
 }
 
+export function getCssFromSheet(sheet) {
+  return [...sheet.renderer.getRules()]
```

Comment:
yeah it fallbacks to Array.from

https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&targets=&browsers=&builtIns=false&debug=false&code=const%20a%20%3D%20%5B...b%5D